changelog/unreleased/enhancement-export-translations

```diff
@@ -0,0 +1,7 @@
+Enhancement: Export translations
+
+Some ODs components depend on translations and they correctly get pulled from Transifex into 
```

Comment:
There's a typo in `ODS` (lower case s) ;-)

Reply:
Updated 🤦🏽‍♂️